apps/cowswap-frontend/src/common/pure/IconSpinner/index.tsx

```diff
@@ -62,24 +62,28 @@ const Wrapper = styled.div<{ size: number; spinnerWidth: number; bgColor?: strin
       transform: rotate(360deg);
     }
   }
-`;
+`
 
-export function IconSpinner({ currency, image, size = 24, children, bgColor, spinnerWidth = 2 }: IconSpinnerProps) {
+export function IconSpinner({
+  currency,
+  image,
+  size = 24,
+  children,
+  bgColor = '--cow-container-bg-01',
```

Comment:
Is it possible to get this value from a const or even better, make these color names an ENUM?

Imagine if we want to pass something else.
How will we know what values are expected?

Reply:
> Imagine if we want to pass something else.
How will we know what values are expected?

I think there we probably need to agree to only always pass CSS color variables? Can you elaborate how an ENUM would add value here?

Reply:
The colors can be defined in a central place, while also being more legible.

For example:

```ts
enum ColorVariables = {
  CONTAINER_BG_01 = '--cow-container-bg-01'
}

type Props = {
  // ...
  bgColor: ColorVariables
}

function Component({
  // ...
  bgColor = CONTAINER_BG_01
}: Props) {
  // ...
```

Note: code is not tested, might not work or need tweaks.

Reply:
Updated!